GFX & Shader Documentation (#472)
fragcolor-xyz/shards: docs/docs/reference/shards/types.md

    [info] [2022-07-22 13:05:25.864] [T-18072] [logging.cpp::55] [mywire] false
    ```

## Sequence

Type **Sequence** is a collection of values that can be accessed directly via indexes (since items are indexed by contiguous integers).

> Also called a vector or an array. Looks exactly like the [**Seq**](#seq) type, but a **Sequence** type's items are accessible by index (and hence accessible randomly). Example of an **Sequence** type would be: `[43 6 1]`.

Reviewer: ```suggestion
It is also called a vector or an array. A **Sequence**'s items are accessible by index. Example of an **Sequence** type would be: `[43 6 1]`.
```
